core/templates/pages/exploration-player-page/services/extract-image-filenames-from-model.service.spec.ts

```diff
@@ -37,9 +38,9 @@ describe('Extracting Image file names in the state service', () => {
   let sof: SkillObjectFactory;
   let ecs: ContextService;
   let ctms: ContentTranslationManagerService;
-  let explorationDict;
-  let ImageFilenamesInExploration;
-  let skillDict;
+  let explorationDict: ExplorationBackendDict;
+  let ImageFilenamesInExploration: { [x: string]: string[] };
```

Comment:
The defined generic `[x: string]: string[];` is enough so I removed the specific strings that I defined and it works just as fine.